hooks/dct-hook.lua

```diff
@@ -410,7 +410,7 @@ function DCTHooks:onMissionLoadEnd()
 	self.mission_start_mt = DCS.getModelTime()
 	self.mission_start_rt = DCS.getRealTime()
 	log.write(facility, log.DEBUG, string.format("mission_time: %f, %s",
-		tostring(self.mission_time), os.date("!%F %R", self.mission_time)))
```

Comment:
If I recall correctly the reason why I put this in was to prevent the server code from applying its local timezone offset onto the mission time. The unit tests actually override the local timezone of the system the tests are running on to be UTC. But this is likely not the case on an actual server. I am ok with requiring the server's timezone to be set to UTC. You might want to run this on an actual server with a non-UTC timezone to see if this changes the reported time.

Reply:
The initial bug was reported on my computer running at a GMT-3 timezone, and the function was setting DCS times 3 hours forward. I think `!` takes a local time and converts it to UTC, causing the confusion.